chat.go

```diff
@@ -47,23 +49,32 @@ type Chat struct {
-	SlowMode                int           `json:"slow_mode_delay,omitempty"`
-	StickerSet              string        `json:"sticker_set_name,omitempty"`
-	CanSetStickerSet        bool          `json:"can_set_sticker_set,omitempty"`
-	LinkedChatID            int64         `json:"linked_chat_id,omitempty"`
-	ChatLocation            *ChatLocation `json:"location,omitempty"`
-	Private                 bool          `json:"has_private_forwards,omitempty"`
-	Protected               bool          `json:"has_protected_content,omitempty"`
-	NoVoiceAndVideo         bool          `json:"has_restricted_voice_and_video_messages"`
-	HiddenMembers           bool          `json:"has_hidden_members,omitempty"`
-	AggressiveAntiSpam      bool          `json:"has_aggressive_anti_spam_enabled,omitempty"`
-	EmojiExpirationUnixtime int64         `json:"emoji_status_expiration_date"`
+	Bio                      string               `json:"bio,omitempty"`
+	Photo                    *ChatPhoto           `json:"photo,omitempty"`
+	Description              string               `json:"description,omitempty"`
+	InviteLink               string               `json:"invite_link,omitempty"`
+	PinnedMessage            *Message             `json:"pinned_message,omitempty"`
+	Permissions              *Rights              `json:"permissions,omitempty"`
+	Reactions                []react.ReactionType `json:"available_reactions"`
+	SlowMode                 int                  `json:"slow_mode_delay,omitempty"`
+	StickerSet               string               `json:"sticker_set_name,omitempty"`
+	CanSetStickerSet         bool                 `json:"can_set_sticker_set,omitempty"`
+	CustomEmojiSetName       string               `json:"custom_emoji_sticker_set_name"`
+	LinkedChatID             int64                `json:"linked_chat_id,omitempty"`
+	ChatLocation             *ChatLocation        `json:"location,omitempty"`
+	Private                  bool                 `json:"has_private_forwards,omitempty"`
+	Protected                bool                 `json:"has_protected_content,omitempty"`
+	NoVoiceAndVideo          bool                 `json:"has_restricted_voice_and_video_messages"`
+	HiddenMembers            bool                 `json:"has_hidden_members,omitempty"`
+	AggressiveAntiSpam       bool                 `json:"has_aggressive_anti_spam_enabled,omitempty"`
+	CustomEmojiID            string               `json:"emoji_status_custom_emoji_id"`
+	EmojiExpirationUnixtime  int64                `json:"emoji_status_expiration_date"`
+	BackgroundEmojiID        string               `json:"background_custom_emoji_id"`
+	AccentColorID            int                  `json:"accent_color_id"`
+	ProfileAccentColorID     int                  `json:"profile_accent_color_id"`
+	ProfileBackgroundEmojiID string               `json:"profile_background_custom_emoji_id"`
+	VisibleHistory           bool                 `json:"has_visible_history"`
```

Comment:
I'm thinking about breaking a contract here and renaming some boolean fields with `has_` prefix, since they're not that clear when used, for instance:

```go
if chat.HiddenMembers {}
if chat.VisibleHistory {}
```
**vs**
```go
if chat.HasHiddenMembers {}
if chat.HasVisibleHistory {}
```

"Chat Has Visible History" is so much better in reading...